components/TopBarV2.js

```diff
@@ -70,6 +71,14 @@ const TopBarV2 = ({ showSearch, menuItems, showProfileAndChangelogMenu }) => {
   const [showSearchModal, setShowSearchModal] = useState(false);
   const ref = useRef();
 
+  useEffect(() => {
+    if (showSearchModal) {
+      NProgress.configure({ parent: '.searchModal' });
```

Comment:
Could this be solved by changing the `zIndex` of NProgress? This looks quite hacky

Reply:
@Betree : Definitely a better approach. 🤔 I've tweaked the NProgress css to handle this. 👍🏽